README.md

```diff
@@ -5,32 +5,34 @@
 
 ## Server
 
-[Mailgun](http://www.mailgun.com/) is used to route email from the `iojs.org` domain to the aliases, as described in this repo.
+[Mailgun](http://www.mailgun.com/) is used to route email from the `iojs.org` domain to the aliases described in this repo.
 
 
 ## Email aliases
 
-The [iojs.org](./iojs.org) directory contains the [aliases.json](./iojs.org/aliases.json) file that describes email aliasing. It is a mapping of `"from"` usernames @iojs.org to `"to"` actual email addresses.
+The [iojs.org](./iojs.org) directory contains the [aliases.json](./iojs.org/aliases.json) file which describes the wanted email aliasing. It is structured as a mapping of `"from"` usernames @iojs.org to `"to"` actual email addresses.
 
 
 ## Updating
 
 ### Responsibility
 
-Merging a PR sould be done concomitantly with updating the mail routes in Mailgun. Since access to the Mailgun API key is required, only members of the @nodejs/build-infra team have the permission to push code to the `master` branch. That is in order to minimize the probability of mismatches between the information in this repo, and the actual email routes setup.
-
-The Mailgun API key can be found in the `admin_logins.md` file in the secrets repo (build/infra/), or direclty via the Rackspace API.
+Changes to the information in this repo sould be done concomitantly with updates to the configured mail routes in Mailgun. Since access to the Mailgun API key is required, only members of the @nodejs/build-infra team have the permission to push code to the `master` branch. That was done in order to minimize the posibility of mismatches between the information in this repo, and the actual email routes that are set up.
```

Comment:
I think the first sentence might make this paragraph less clear than if it weren't there. Perhaps remove it

Reply:
Ack